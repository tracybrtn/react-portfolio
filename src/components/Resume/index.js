import React from "react";
import ResumePdf from "../../../src/assets/files/TRACY BURTON BRAVO - RESUME.pdf";

function Resume(){
  return(

  <section>
    <h1 id="resume">Resume</h1>
    <a href={ResumePdf} download>Download my resume</a>
    <h2>Developer Proficiencies</h2>
    <ul>
      <li>HTML</li>
      <li>CSS</li>
      <li>JS</li>
      <li>node.js</li>
      <li>State</li>
    </ul>
  </section>
  );
}

export default Resume;