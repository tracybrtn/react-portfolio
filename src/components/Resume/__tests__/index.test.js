import React from 'react';
import { render, cleanup } from '@testing-library/react';
import '@testing-library/jest-dom/extend-expect';
import Resume from '..';


afterEach(cleanup);

describe('Contact component', () => {
  // baseline test
  it('renders', () => {
    render(<Resume/>);
  });

  //snapshot test
  it('matches snapshot', () => {
    const { asFragment } = render(<Resume />);

    expect(asFragment()).toMatchSnapshot();
  });
})