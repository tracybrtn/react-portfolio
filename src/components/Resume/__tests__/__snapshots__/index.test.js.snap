// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Contact component matches snapshot 1`] = `
<DocumentFragment>
  <section>
    <h1
      id="resume"
    >
      Resume
    </h1>
    <a
      download=""
      href="TRACY BURTON BRAVO - RESUME.pdf"
    >
      Download my resume
    </a>
    <h2>
      Developer Proficiencies
    </h2>
    <ul>
      <li>
        HTML
      </li>
      <li>
        CSS
      </li>
      <li>
        JS
      </li>
      <li>
        node.js
      </li>
      <li>
        State
      </li>
    </ul>
  </section>
</DocumentFragment>
`;
